sdk/azcore/policy_anonymous_credential_test.go

```diff
@@ -17,8 +17,8 @@ func TestAnonymousCredential(t *testing.T) {
 	defer close()
 	srv.SetResponse(mock.WithStatusCode(http.StatusOK))
 	pl := NewPipeline(srv, AnonymousCredential().AuthenticationPolicy(AuthenticationPolicyOptions{}))
-	req := pl.NewRequest(http.MethodGet, srv.URL())
-	resp, err := req.Do(context.Background())
+	req := NewRequest(http.MethodGet, srv.URL())
```

Comment:
Are we leaving this as a publicly exported function?

Reply:
Yep!  We no longer have to create a request via a pipeline, just create it directly.